docker/run/Dockerfile

```diff
@@ -50,7 +51,9 @@ ENV \
     # Installation directories.
     FLUTTER_HOME=/opt/flutter
 
-ENV PATH="$PATH:$FLUTTER_HOME/bin:$FLUTTER_HOME/bin/cache/dart-sdk/bin"
+ENV \
+    PATH="$PATH:$FLUTTER_HOME/bin:$FLUTTER_HOME/bin/cache/dart-sdk/bin:/opt/go/bin" \
+    GOPATH="/go"
```

Comment:
What's the reason to not keep the [default](https://github.com/golang/go/wiki/SettingGOPATH#unix-systems) of `$HOME/go` here?

Reply:
I understand you use the term `default` for the recommendation in the linked installation instruction. I think those instructions are likely to target primarily the installation on typical multi user systems rather than in a docker image. I don't see the reason why one should need to align exactly with those instructions. Do you see any?

I'd rather see our `docker` image lack the `user` related bits and instead get those ones settled out first before moving anything underneath the `/root` directory. Does this make sense?

Reply:
Several things here: First of all, I'm wonder why need to set this explicitly at all, as there is an implied default. If we'd want to use something else than the default, I'd also like to understand why, esp. if this non-default value is a global directory like `/go`.

Secondly, our Docker image is special in that [we do not run it as root](https://github.com/heremaps/oss-review-toolkit/blob/17da95aab915de32df6f2c2da6ba62b46805ddcc/docker/lib#L29-L41). As a result, we should not store user data in non-user directories.

Reply:
Interresting, wasn't aware of that second thing...thanks!